Query optimization

codingotaku opened the pull request:
Separated Queries in IndexContent.vue
Send index list along with platform to avoid client side processing *this needs a little more change for memory efficiency*

- Added Pagination for node table - *This significantly reduces load time*
- Added Search feature for nodes - now nodes are searchable by name

This should close #213 and #176 
Potentially avoiding caching mentioned in #187

### TODO

- Make pagination user-friendly
  - Currently jumping to a specific page is not supported
  - Look and feel of pagination is very ugly
- Remove Redundant code for pagination (these can be moved to NodesTable.vue)
- Decide whether to apply filter for status when filtering NodesTable

jaywink commented on src/components/NodesContent.vue:
```diff
@@ -59,30 +78,92 @@ const query = gql`
+            }
+            this.getPage(this.page + 1)
+        },
+        getPreviosuPage() {
```
Typo

codingotaku replied:
woops! Will fix it

codingotaku replied:
Done

jaywink reviewed (commented):
Nothing to add to this :+1:  Looks great to me and if it takes things to the direction of a faster site then :tada:  Thanks for working on this. Lets get it on a staging side asap, I'll get it deployed in the next few days and let you know :+1:

codingotaku commented:
Is now cleaned up, removed redundant codes and fixed typos.

@jaywink the stats are not showing up for me - `statsGlobalToday` is null and `statsNodes` is empty - it's probably because my database has old data. If you are seeing the same issue when using live db, let me know.

codingotaku commented:
Ah... my bad, the change breaks nodes page... It's an easy fix will do soon.
